recipes/apache-solr/meta.yaml

```diff
@@ -0,0 +1,37 @@
+{% set name = "apache-solr" %}
```

Comment:
probably don't  need `name` variable, as it's only ever used once

Reply:
True - fair point.